jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java

```diff
@@ -208,32 +203,26 @@ private static void initializeJettyLogging() {
     private static void setupJmx(Server server, ServerConfig serverConfig) {
         if (serverConfig.jmx().enabled()) {
             System.setProperty("java.rmi.server.hostname", "localhost");
-            server.addBean(
-                    new MBeanContainer(ManagementFactory.getPlatformMBeanServer()));
-            server.addBean(
-                    new ConnectorServer(
-                            createJmxLoopbackOnlyServiceUrl(serverConfig.jmx().listenPort()),
-                            "org.eclipse.jetty.jmx:name=rmiconnectorserver"));
+            server.addBean(new MBeanContainer(ManagementFactory.getPlatformMBeanServer()));
+            server.addBean(new ConnectorServer(createJmxLoopbackOnlyServiceUrl(serverConfig.jmx().listenPort()),
+                                               "org.eclipse.jetty.jmx:name=rmiconnectorserver"));
         }
     }
 
     private static JMXServiceURL createJmxLoopbackOnlyServiceUrl(int port) {
         try {
-            return new JMXServiceURL(
-                    "rmi", "localhost", port, "/jndi/rmi://localhost:" + port + "/jmxrmi");
+            return new JMXServiceURL("rmi", "localhost", port, "/jndi/rmi://localhost:" + port + "/jmxrmi");
         } catch (MalformedURLException e) {
             throw new RuntimeException(e);
         }
     }
 
-    private HandlerCollection getHandlerCollection(
-            ServerConfig serverConfig,
-            ServletPathsConfig servletPathsConfig,
-            List<JDiscServerConnector> connectors,
-            ServletHolder jdiscServlet,
-            ComponentRegistry<ServletHolder> servletHolders,
-            FilterHolder jDiscFilterInvokerFilter) {
-
+    private HandlerCollection getHandlerCollection(ServerConfig serverConfig,
```

Comment:
I actually prefer the previous indentation style. The new style requires a re-indentation of all parameters if the name or return type of the method is changed.

Reply:
But the IDE does that for you automatically ...

Reply:
Not mine, probably missing some non-default configuration.